### PR TITLE
Update to 2.5

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by goanpeca** [comment]
Hi @jjhelmus. It seems the latest anaconda package for pylint did not add the `toml` dependency in the run section but on the tests section.

https://github.com/AnacondaRecipes/pylint-feedstock/blob/master/recipe/meta.yaml#L36

This is breaking Pylint and Spyder on running the `conda update` command

Do you think you could issue a fix :-) ?

Thanks for the help!

**Comment by goanpeca** [comment]
@conda-forge-admin, please re-render.

**Comment by goanpeca** [comment]
@conda-forge-admin, please re-render

**Comment by ccordoba12** [comment]
@jjhelmus, this release of Pylint also requires the latest version of astroid.

**Comment by CAM-Gerlach** [comment]
@jjhelmus Hey, just wanted to give this a bump; just in the past 3 days we've had nearly a dozen users report this just on our issue tracker (I even ran into it myself). Its an easy fix for users who see and understand the error and know what to do about it, but for the large proportion of our userbase that is still fairly new to Python and Anaconda it can be a frustrating experience indeed. Thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,11 +25,12 @@ requirements:
...
   run:
-    - python
-    - astroid >=2.3.0,<2.4
+    - astroid >=2.4.0,<2.5
+    - colorama  # [win]
     - isort >=4.2.5,<5
     - mccabe >=0.6,<0.7
-    - colorama  # [win]
+    - python
+    - toml
```
Comment (jjhelmus) [inline recipe/meta.yaml:33]:
```suggestion
    - toml >=0.7.1
```

**Comment by jjhelmus** [comment]
Upstream has a minimum bound on toml of >=0.7.1, can we add that here?

**Comment by jjhelmus** [comment]
> Hi @jjhelmus. It seems the latest anaconda package for pylint did not add the `toml` dependency in the run section but on the tests section.

There are a few things wrong with that package.  I'll get a new build out and figure out the best opinion for the existing package.

**Comment by ccordoba12** [comment]
Thanks a lot @jjhelmus for your help!

**Comment by jjhelmus** [comment]
Merging, Travis CI is failing because the ppc64le machines provided by that service are broken.  This should not hold up the other platforms.

**Comment by jjhelmus** [comment]
> There are a few things wrong with that package. I'll get a new build out and figure out the best opinion for the existing package.

The build 1 pylint 2.5.0 packages in `defaults` now have the correct dependency information.  In addition, I've hotfixed (AnacondaRecipes/repodata-hotfixes#82) the build 0 packages to correct the missing toml requirement and modify the bounds on `astroid`.  
Updated or new environments will get the correct requirements.

**Comment by CAM-Gerlach** [comment]
Thanks so much @jjhelmus !

**Comment by goanpeca** [comment]
Thanks for the help @jjhelmus !

**Pull request merged**